projects/igniteui-angular/src/lib/grids/summaries/summary-cell.component.html

```diff
@@ -7,21 +7,9 @@
             <div class="igx-grid-summary__item" [style.height.px]="itemHeight">
 
                 <span class="igx-grid-summary__label" [title]="summary.label">{{ translateSummary(summary) }}</span>
-                <span class="igx-grid-summary__result" [title]="summary.summaryResult">
+                <span class="igx-grid-summary__result" [title]="formatSummaryResult(summary)">
                     {{
-                        summaryFormatter
-                        ? (summary | summaryFormatter:column.summaries:summaryFormatter)
-                        : (isNumberColumn() && summary.defaultFormatting)
-                        ? (summary.summaryResult | number:column.pipeArgs.digitsInfo:grid.locale)
-                        : (isDateKindColumn() && summary.defaultFormatting)
-                        ? (summary.summaryResult | date:column.pipeArgs.format:column.pipeArgs.timezone:grid.locale)
-                        : (isCurrencyColumn() && summary.defaultFormatting)
-                        ? (summary.summaryResult | currency:currencyCode:column.pipeArgs.display:column.pipeArgs.digitsInfo:grid.locale)
-                        : (isPercentColumn() && summary.defaultFormatting)
-                        ? (summary.summaryResult | percent:column.pipeArgs.digitsInfo:grid.locale)
-                        : (summary.key === 'count')
-                        ? (summary.summaryResult | number:undefined:grid.locale)
-                        : summary.summaryResult
+                        formatSummaryResult(summary)
                     }}
```

Comment:
An alternative solution would be to simply copy this statement as the title attribute's value, which is also more straigtforward, however repetitive. @hanastasov, which would be better in this case?
Additionally, with the current approach, the `is*Type*Column()` methods in  the `IgxSummaryCellComponent` are left redundant.

Reply:
This approach is just fine. Good job! Much more readable now. I think you can remove the IsTypeColumn methods.